clippy.toml

```diff
@@ -0,0 +1,3 @@
+disallowed-types = [
+    { path = "std::collections::HashMap", reason = "HashMap's key order is unspecified and thus serializing such a type will result in a random output, use BTreeMap instead." }
```

Comment:
I put that long `reason` in a single line as newline is [still not allowed in `inline table`](https://github.com/toml-lang/toml/blob/913ff560d935e7fb2788b402abba567680f6e03c/CHANGELOG.md#unreleased)

If we choose to use multiple line string, it would look like this:

```tom
disallowed-types = [
    { path = "std::collections::HashMap", reason = """
      HashMap's key order is unspecified and thus serializing such a type will
      result in a random output, use BTreeMap instead.
      """ }
]
```
which is also not good:(